test/study-webidl.js

```diff
@@ -56,8 +77,8 @@ interface Valid: Invalid {};
 interface Invalid{};
 `);
     const report = await studyWebIdl(crawlResult, curatedResult);
-    assert.deepEqual(report.length, 1);
-    assert.deepEqual(report[0]?.name, 'invalid');
+    assertEqual(report.length, 1);
```

Comment:
```suggestion
    assert.equal(report.length, 1);
```
I don't even want to think how this doesn't fail atm :)